crucible-mir/src/Mir/TransTy.hs

```diff
@@ -314,6 +315,26 @@ isZeroSized col ty = go ty
       _ -> False
 
 
+-- | Get the "ABI-level" function arguments for `sig`, which determines the
+-- arguments we use for the Crucible `FnHandle`.  This applies a special
+-- adjustment for `extern "rust-call"` functions, like the one rustc applies
+-- when converting a `FnSig` to `FnAbi`.
```

Comment:
Perhaps state that the "special adjustment" is untupling?

Reply:
I adjusted this comment slightly in 35f00b44baee28f5649bba773df875c95d3b71be.  I don't think it's necessary to be specific about the adjustment - the point is that this function "does the right thing" in all cases, including `extern "rust-call"`.